src/airflow/airqo_etl_utils/airnow_utils.py

```diff
@@ -65,8 +64,15 @@ def extract_bam_data(start_date_time: str, end_date_time: str) -> pd.DataFrame:
     def process_bam_data(data: pd.DataFrame) -> pd.DataFrame:
 
         air_now_data = []
+
+        devices = AirQoApi().get_tenant_devices(tenant=Tenant.US_EMBASSY)
```

Comment:
I am somehow struggling to juggle the "tenant" with "network"....I hope to get more alignment since we are now targeting one DB (tenant) in device-registry.

Otherwise, no worries in case the alignment is straight forward from your end.